README.md

```diff
@@ -44,7 +44,7 @@ Use:
 
 If you prefer not to use the plugin you can also import the component directly:
 ```javascript
-import ToggleButton from 'vue-js-toggle-button/src/Button'
+import ToggleButton from 'vue-js-toggle-button/src/Button.vue''
```

Comment:
You don't have to specify it as far as I remember (unless your webpack  vue-loader is slightly outdated) also there is an extra quote symbol

Reply:
It was broken without the .vue at the end.